src/applications/personalization/dashboard/utils/claims-helpers.js

```diff
@@ -236,6 +236,26 @@ export function isClaimComplete(claim) {
   return claim.attributes.decisionLetterSent || claim.attributes.phase === 8;
 }
 
+export function isLighthouseClaimComplete(claim) {
+  return (
+    claim.attributes.decisionLetterSent ||
+    claim.attributes.status === 'COMPLETE'
+  );
+}
+
+const lightHouseClaimStatusMap = {
+  CLAIM_RECEIVED: 'Claim received',
+  INITIAL_REVIEW: 'Initial review',
+  EVIDENCE_GATHERING_REVIEW_DECISION:
+    'Evidence gathering, review, and decision',
+  PREPARATION_FOR_NOTIFICATION: 'Preparation for notification',
+  COMPLETE: 'Closed',
+};
+
+export function getLighthouseClaimStatusDescription(status) {
+  return lightHouseClaimStatusMap[status];
+}
+
```

Comment:
It may be an impossible edge case, but just thought I would bring it up. 

The thought also came to mind of moving things like this to the `/constants` directory of the dashboard. Might help clean up this file a bit. There are a lot of what looks to be static values in this helpers file. Could be a future ticket to reduce a little tech debt if yall think its a good idea. Thoughts @dellerbie and @allisonlu?